pages/docs/data-deletion.mdx

```diff
@@ -0,0 +1,74 @@
+---
+description: Delete data from Langfuse
+---
+
+# Data Deletion
+
+<AvailabilityBanner
+  availability={{
+    hobby: "full",
+    pro: "full",
+    team: "full",
+    selfHosted: "full",
+  }}
+/>
+
+There may be use-cases where you want to remove selected data from Langfuse, like erroneously created traces in a development flow, user data for PII, or your whole project.
+In case you want to retain only recent data, you can use our [Data Retention](/docs/data-retention) feature.
+
+You can delete unwanted data from Langfuse by:
+
+- Deleting a single trace;
+- Deleting a batch of traces;
+- Deleting all traces that match a query filter;
+- Deleting a project; or
+- Deleting an organization.
+
+Below, we will walk through each of the options and their guarantees.
+
+## Deleting Traces
+
+### Single Trace
+
+To delete a single trace, open its detail view and hit the `Delete` button.
+Confirm that you want to delete the given trace.
+
+![Delete a single trace](/images/docs/delete-single-trace.png)
+
+### Batch of Traces
+
+To delete a batch of traces, select them in the trace list and select `Delete` in the `Actions` dropdown.
+
+![Delete a batch of traces](/images/docs/delete-trace-batch.png)
+
+### Delete by Query
+
+To delete all traces that match a query filter, configure your desired filter in the traces list.
+Select all items on the current page and change that to all items in the top bar.
+Then select `Delete` in the `Actions` dropdown.
+
+![Delete traces by query](/images/docs/delete-filtered-traces.png)
+
+### Limitations
+
+While most deletions in Langfuse happen instantly, event data like traces is not.
+Removing those records from our data warehouse is a resource intensive operation and, therefore, we rate limit
+how many deletions we process at any point in time.
+Usually, trace data is deleted from our system within 15 minutes of the delete call and there is no confirmation, i.e.
+in order to verify that your data got deleted, you will have to query it again.
+
+## Deleting a Project
+
+To delete a project, navigate to the project settings and scroll to the `Danger Zone` within the `General` section.
+Confirm that you want to delete your project.
+This action immediately revokes all API keys and schedules the project for deletion.
+Within the next minutes, all related data is irreversibly removed from our system.
+
+<Callout type="warning">
+  Deleting a project is irreversible and all data will be removed.
+  Be cautious when executing this action.
+</Callout>
+
+## Deleting an Organization (Cloud only)
+
+To delete an organization, please message us at support@langfuse.com.
```

Comment:
Orgs can be deleted when there are no projects within the org anymore. No need to reach out to us. Users need to reach out if they want to delete their user account.

![CleanShot 2025-02-18 at 12 54 47](https://github.com/user-attachments/assets/b875d814-a51c-40ac-b6d2-06d111a20fff)

Reply:
@marcklingen Thank you for the feedback. Updated via https://github.com/langfuse/langfuse-docs/pull/1287/commits/d3cfa7b36cef399224ed62859fa07287645f9b64.